packages/maskbook/src/extension/options-page/DashboardComponents/Drawer.tsx

```diff
@@ -125,7 +125,8 @@ export default function Drawer(props: DrawerProps) {
 
     const { routers } = props
 
-    const openFeedback = () => {
+    const onFeedback = () => {
+        // see https://github.com/DimensionDev/Maskbook/issues/3384
```

Comment:
```suggestion
        // see #3384
```

Reply:
I don't think so because people can't directly open the link by `Ctrl + left click.`